shared/src/business/entities/DocketEntry.ts

```diff
@@ -466,15 +480,194 @@ export class DocketEntry extends JoiValidationEntity {
   }
 
   isCourtIssued(): boolean {
-    return DocketEntry.isCourtIssued(this.eventCode);
+    return DocketEntry.isCourtIssued({ eventCode: this.eventCode });
   }
 
-  static isCourtIssued(eventCode: string): boolean {
+  static TRANSCRIPT_AGE_DAYS_MIN = 90;
+
+  static isCourtIssued({ eventCode }: { eventCode: string }): boolean {
     return COURT_ISSUED_EVENT_CODES.map(
       ({ eventCode: courtIssuedEventCode }) => courtIssuedEventCode,
     ).includes(eventCode);
   }
 
+  static isAvailableToExternal = (entry: RawDocketEntry): boolean => {
```

Comment:
To make sure I'm thinking the same thing, 'external' here means someone who is logged in to DAWSON but does not work for the court?

Reply:
External = non Court User